src/app.css

```diff
@@ -0,0 +1,214 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.nav {
+  width: 100%;
+  height: 80px;
+  display: flex;
+  flex-direction: row;
+  background-color: #f1f1f1;
+}
+
+.nav-main {
+  display: flex;
+  width: 95%;
+  margin: 0 auto;
+}
+
+.logo {
+  width: 20%;
+}
+
+.logo img {
+  width: 85px;
+  height: auto;
+}
+
+.menu {
+  width: 55%;
+  display: inline-block;
+  text-align: center;
+}
+
+.menu ul li {
+  display: inline-block;
+  margin: 30px 15px;
+  vertical-align: middle;
+}
+
+.menu ul li a, .m-menu li a {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".m-menu li a" to come before selector ".menu ul li a" (no-descending-specificity)](https://app.codacy.com/gh/BuildForSDG/EduKolab-Frontend/pullRequest?prid=5585842)